bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/LinkyBindingConstants.java

```diff
@@ -32,16 +36,49 @@ public class LinkyBindingConstants {
     // Thing properties
     public static final String PUISSANCE = "puissance";
     public static final String PRM_ID = "prmId";
-    public static final String USER_ID = "av2_interne_id";
+    public static final String USER_ID = "customerId";
 
     // List of all Channel id's
-    public static final String YESTERDAY = "daily#yesterday";
-    public static final String PEAK_POWER = "daily#power";
-    public static final String PEAK_TIMESTAMP = "daily#timestamp";
-    public static final String THIS_WEEK = "weekly#thisWeek";
-    public static final String LAST_WEEK = "weekly#lastWeek";
-    public static final String THIS_MONTH = "monthly#thisMonth";
-    public static final String LAST_MONTH = "monthly#lastMonth";
-    public static final String THIS_YEAR = "yearly#thisYear";
-    public static final String LAST_YEAR = "yearly#lastYear";
+    public static final String DAY_MINUS_1 = "daily#day-1";
```

Comment:
I would avoid breaking compatibility on channels if it can be avoided. 
This will make it tougher to have it merged (while I agree your naming convention is cleaner).

If you need to do so, do it in a second PR (let say when you'll add new functionalities allowed by this API change) and then you'll have to provide migration instructions for the channel name changes so they are automatically picked up and applied for managed items.

Reply:
I realign them to use the old naming for now.
Perhaps a much cleaner approach for next iteration would be to introduce a new serie of Channel that will use the new TimeSeries capabilities.

I will review this later.

I also introduce a bunch of new channel to get Enedis Contract information like subscription power, contractor information, offOn Peak hours and others...